test/test_gpt2tok.py

```diff
@@ -23,90 +23,103 @@ def _get_test_data_file(*sub_dirs):
+            return input_ids, attention_mask
+
+    def _run_tokenizer(self, test_sentence, padding_length=-1):
+        model = _create_test_model(vocab_file=self.tokjson, merges_file=self.merges, max_length=padding_length)
         so = _ort.SessionOptions()
         so.register_custom_ops_library(_get_library_path())
-        sess = _ort.InferenceSession(self.binded_model.SerializeToString(), so)
-        input_text = np.array([test_sentence])
-        txtout = sess.run(None, {'string_input': input_text})
+        sess = _ort.InferenceSession(model.SerializeToString(), so)
+        input_text = np.array(test_sentence)
+        input_ids, attention_mask = sess.run(None, {'string_input': input_text})
+        expect_input_ids, expect_attention_mask = self.tokenizer.tokenizer_sentence(test_sentence, padding_length)
+        np.testing.assert_array_equal(expect_input_ids, input_ids)
+        np.testing.assert_array_equal(expect_attention_mask, attention_mask)
 
-        np.testing.assert_array_equal(txtout[0], np.array([self.tokenizer.encode(test_sentence)]))
         del sess
         del so
 
     def test_tokenizer(self):
-        TestGPT2Tokenizer.pyop_invoked = False
-
-        self._run_tokenizer(False, "I can feel the magic, can you?")
-        self.assertFalse(TestGPT2Tokenizer.pyop_invoked)
-
-        self._run_tokenizer(False, "Hey Cortana")
-        self._run_tokenizer(False, "你好123。david")
-        self._run_tokenizer(False, "women'thinsulate 3 button leather car co")
-        self._run_tokenizer(False, "#$%^&()!@?><L:{}\\[];',./`ǠǡǢǣǤǥǦǧǨ")
-        self._run_tokenizer(False, "ڠڡڢڣڤڥڦڧڨکڪګڬڭڮگ")
-        self._run_tokenizer(False, "⛀⛁⛂⛃⛄⛅⛆⛇⛈⛉⛊⛋⛌⛍⛎⛏")
-
-    def test_tokenizer_pyop(self):
-        TestGPT2Tokenizer.pyop_invoked = False
-        self._run_tokenizer(True, "I can feel the magic, can you?")
-        self.assertTrue(TestGPT2Tokenizer.pyop_invoked)
-
-        self._run_tokenizer(True, "Hey Cortana")
-        self._run_tokenizer(True, "你好123。david")
-        self._run_tokenizer(True, "women'thinsulate 3 button leather car co")
-        self._run_tokenizer(True, "#$%^&()!@?><L:{}\\[];',./`ǠǡǢǣǤǥǦǧǨ")
-        self._run_tokenizer(True, "ڠڡڢڣڤڥڦڧڨکڪګڬڭڮگ")
-        self._run_tokenizer(True, "⛀⛁⛂⛃⛄⛅⛆⛇⛈⛉⛊⛋⛌⛍⛎⛏")
+        enable_custom_op(False)
+
+        self._run_tokenizer(["I can feel the magic, can you?"])
+        self._run_tokenizer(["Hey Cortana"])
+        self._run_tokenizer(["你好123。david"])
+        self._run_tokenizer(["爱你一三一四"])
+        self._run_tokenizer(["women'thinsulate 3 button leather car co"])
+        self._run_tokenizer(["#$%^&()!@?><L:{}\\[];',./`ǠǡǢǣǤǥǦǧǨ"])
+        self._run_tokenizer(["ڠڡڢڣڤڥڦڧڨکڪګڬڭڮگ"])
+        self._run_tokenizer(["⛀⛁⛂⛃⛄⛅⛆⛇⛈⛉⛊⛋⛌⛍⛎⛏"])
+        self._run_tokenizer(["I can feel the magic, can you?", "Yes I do."])
+        self._run_tokenizer(["I can feel the magic, can you?", "Yes I do."], 100)
+
+        enable_custom_op(True)
+
+
+# def test_tokenizer_pyop(self):
```

Comment:
No pyop support any more?

Reply:
No, it's just disabled temporarily,  due to #74